applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_prebuckling_analysis.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics
+# Importing the base class
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis
+
+class StructuralMechanicsPrebucklingAnalysis(StructuralMechanicsAnalysis):
```

Comment:
I derived the analysis stage, because I don't know in advance how many load steps the solution will need.  I basically use the "end_time" as maximal load iteration.
So the RunSolutionLoop is overriden to break, when the solution is converged. Are there better options to do this?
Futhermore, I only want to postprocess the eigenvalues in every second load step, because the eigenvalue problem is only solved every second step.

Reply:
@philbucher is this ok?